lib/hooks/core/hook-http.js

```diff
@@ -54,19 +53,21 @@ function requestWrap(request) {
       return request.apply(this, arguments);
     }
 
+    var localOptions = (typeof options === 'object') ?
```

Comment:
Did you miss my other comment? Get rid of the `localOptions` variable and assign to `options`.